app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -32,6 +34,8 @@ class MusicsListFragment : Fragment() {
     @Inject
     lateinit var viewModelFactory: ViewModelFactory
 
+    private var musicList = ArrayList<Music>()
```

Comment:
toye fragment?!!!

Reply:
movaghat gozashtam

Reply:
khob key eslah mishe?
PR merge she nemishe taghyiresh dad.
va benazaram ravande aslie app bayad ghabl az merge in PR kamle va dorost bashe